tests/integration/targets/aws_glue_job/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+role_name: '{{ resource_prefix }}-role'
```

Comment:
`role_name` is a reserved keyword, it resolves to the name of the test role (in this case, `aws_glue_job`).  Could you please change this to something else?  Something like `glue_role_name` or `test_role` are fairly common patterns in other tests.